apps/fishing-map-e2e/cypress.config.ts

```diff
@@ -8,7 +8,7 @@ const cypressJsonConfig = {
   videosFolder: '../../dist/cypress/apps/fishing-map-e2e/videos',
   screenshotsFolder: '../../dist/cypress/apps/fishing-map-e2e/screenshots',
   chromeWebSecurity: false,
-  specPattern: 'src/e2e/**/*.cy.{js,jsx,ts,tsx}',
+  specPattern: ['src/e2e/app.cy.ts', 'src/e2e/login.cy.ts', 'src/e2e/**/*.cy.{js,jsx,ts,tsx}'],
```

Comment:
@nataliafz do we need to define every test case file here?

Reply:
@j8seangel no, with 'src/e2e/**/*.cy.{js,jsx,ts,tsx}' should be enough

Reply:
I'd remove the fixed ones then to avoid confusion

Reply:
without that won't work @j8seangel @rdgfuentes, the reason is that in login we are removing a popup and then in vms we dont need to remove it again because the localstorage is not shown again, if the e2e modify this order the popup appear and the test fail. 
For this I implemented a function in another test that hide all the popups so then I just need to include that in the vms test.
But also my idea is to avoid tests when one fail, so the idea is just to test if the app open, and maybe other basic tests, before the more complex tests that can take more time